charts/datadog/values.yaml

```diff
@@ -1183,9 +1183,18 @@ fips:
     # fips.image.digest -- Define the FIPS sidecar image digest to use, takes precedence over `fips.image.tag` if specified.
     digest: ""
 
-    # agents.image.repository -- Override default registry + image.name for Agent
+    # fips.image.repository -- Override default registry + image.name for the FIPS sidecar container.
     repository:
 
+  # fips.useConfigMap -- Configures a configmap to provide the FIPS configuration. Use this in combination with the `fips.customFipsConfig` parameter.
+  useConfigMap:  # false
```

Comment:
Can we use a single config option for this instead of two separate ones? I would assume that having a config map would imply `useConfigMap`.

Reply:
Given the fact that the users may easily fall out of compliance using these option, I think we also need this caveat noted explicitly in the comments around them.

Reply:
Great idea, updated: https://github.com/DataDog/helm-charts/pull/1095/files#diff-2fcec98016e714405bc936fd949012c2e72f23c20a1a94acf695a9cd3e29d75fR1185-R1192